### DISCUSSION TITLE
How can I "strip" in weapons.cfg to remove all attributes from a weapon?

### DISCUSSION THREAD

**Discussion by komisz01** [Batfoxkid/Freak-Fortress-2-Rewrite#207] [Q&A]
I'm having problems in FF2 rewrite weapons config.
I want to replace the jumper weapons, Sticky Jumper and Rocket Jumper to default weapons, Sticky Launcher and Rocket Launcher because I think they're OP in the game mode.
For that, I tried using the "strip", but it didn't look like that it changed something on the Jumping weapons, like It didn't changed anything on it, they working the same, didn't removed the attributes from the weapon at all.
Did I typed something wrong in the config? Please let me know.

"""
"Indexes"
{
    "237"    // Rocket Jumper
    {
        "strip"    "true"
    }
    "265"    // Sticky Jumper
    {
        "strip"    "true"
    }
}
"""

**Comment by doroemon** [comment]
try this
```
"237"	// Rocket Jumper
{
	"attributes"	"181 ; 0 ; 1 ; 1.0 ; 15 ; 1.0 ; 76 ; 1.0"
}
"265"	// Sticky Jumper
{
	"attributes"	"181 ; 0 ; 1 ; 1.0 ; 15 ; 1.0 ; 78 ; 1.0"
}
```

**Reply by komisz01** [reply]
It's working. Thank you!

**Comment by TimqReal** [comment]
I have the same problem, i guess strip no longer works
